tests/test_onnx/utils.py

```diff
@@ -32,14 +33,22 @@ def ort_validate(model, feats, onnx_io='tmp.onnx'):
     generated by torch.
 
     Args:
-        model (nn.Module): the model to be verified
-        feats (list(torch.Tensor) | torch.Tensor): the input of model
-        onnx_io (str): the name of onnx output file
+        model (nn.Module | function) : the function of model or model
+            to be verified.
+        feats (tuple(list(torch.Tensor)) | list(torch.Tensor) | torch.Tensor):
+            the input of model.
+        onnx_io (str): the name of onnx output file.
     """
-    model.cpu().eval()
+    # if mdoel is not a nn.Module, then it is a function and wrapped
```

Comment:
if model is not an instance of nn.Module, then it is a normal function and it should be wrapped.